common/bytes_test.go

```diff
@@ -103,3 +103,55 @@ func TestNoPrefixShortHexOddLength(t *testing.T) {
 		t.Errorf("Expected %x got %x", expected, result)
 	}
 }
+
+func TestInt64ToByteBigEndian(t *testing.T) {
+	expected := []byte{1, 3, 4, 2}
```

Comment:
```suggestion
	expected := []byte{9, 1, 5, 3, 4, 2, 8, 6}
```

Reply:
Review reflected. Thank you for pointing it out!